docs/build/collator/how-to-stake-in-zeitgeist.md

```diff
@@ -59,6 +59,16 @@ console.log(delegatorInfo.toHuman()["delegations"].length);
 
 ![](/img/get-your-delegations-number.png)
 
+### Check the amount you can stake
+
+You have a balance which consists of transferrable, vested, locked and reserved tokens. To see the amount of stakable tokens, you can check the transferable tokens on the ` Account` Page
+
+1.Choose ` Account`
+
+2.Click the triangle button
```

Comment:
A bit more specific description might be helpful here, for example: "Click the downward pointing triangle button to the right side of the Account you want to know the balances of". Ideally "downward pointing triangle" will be represented as the icon.